packages/faucet/request_test.go

```diff
@@ -86,10 +88,11 @@ func TestIsFaucetReq(t *testing.T) {
 	)
 
 	dataMsg, _ := tangle.NewMessage(
-		tangle.MessageIDs{tangle.EmptyMessageID},
-		tangle.MessageIDs{},
-		tangle.MessageIDs{},
-		tangle.MessageIDs{},
+		map[tangle.ParentsType]tangle.MessageIDs{
+			tangle.StrongParentType: {
+				tangle.EmptyMessageID: types.Void,
+			},
```

Comment:
This could be massively simplified by introducing a type and functional options.